docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -12,7 +12,7 @@ install them yourself.
 1. Install `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
 1. Install `kustomize`, refer to official instructions [here](https://kubectl.docs.kubernetes.io/installation/kustomize/).
 1. Install Ironic, refer to [this page](../ironic/ironic_installation.html).
-1. Install Baremetal Operator, refer to [TODO](TODO).
+1. Install Baremetal Operator, refer to [this page](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/bmo/install_baremetal_operator.md).
```

Comment:
The link should be relative, just like the ironic installation link is above. `../bmo/install_baremetal_operator.html` or so. We do not point to source files of the book in the book, but to generated content.

You also need to rebase this commit, so the BMO installation guide merged in #352 is visible in your preview as well.

Reply:
@tuminoid  I have reviewed the link and also rebased the commits could you please review and offer feedback? thank you

Reply:
You're now including the content from #352 in your commit. You need to rebase, something like this:
- `git fetch --all`
- `get rebase origin/source`
- resolve any  possible conflicts, then `git add <files>`, `git rebase --continue` (skip all if no conflicts)
- git push your branch into your fork

This PR must only contain one line diff where you fix the link.

Reply:
You can see here https://github.com/metal3-io/metal3-docs/pull/351/files that you're still including more than the link.

Reply:
yes sir I just figured out that was the problem, I tried the processes you gave me but the files are still there, when i run git rebase origin/main it tells me everything is up to date. it seems the files are embedded in my commit. I'm currently looking for a way to remove those files from my commit

Reply:
I can't tell you the exact commands to run as I don't know your exact setup, but the idea is to 'git fetch` changes from the this repository (not your fork) and then `git rebase` your fork versus that fetch head. This would remove the parts already in this repository from your PR. If Git tells you `already up to date`, it means you haven't fetched the upstream repo (this repo) or you're rebasing against wrong remote. 

Something like this: https://medium.com/@ruthmpardee/git-fork-workflow-using-rebase-587a144be470

Reply:
thanks alot @tuminoid the article made it super easy, i really appreciate. it's been fixed.

Reply:
Looks correct now. :tada: